examples/dynamics/body_landing_speed.py

```diff
@@ -0,0 +1,31 @@
+from sympy import solve, Eq
+from symplyphysics import print_expression, units, Symbol
+
+from symplyphysics.laws.dynamics import potential_energy_from_mass_and_height as potential_energy
+from symplyphysics.laws.dynamics import kinetic_energy_from_mass_and_velocity as kinetic_energy
+
+# From http://kornev-school.ru/f9_law_of_energy_conservation.html first example
+# A soccer ball falls without initial velocity from a height.
+# Find its speed before hitting the ground.
+
+height = Symbol("height", units.length)
+gravity_acceleration = Symbol("gravity_acceleration", units.acceleration)
+body_mass = Symbol("mass_of_the_body", units.mass)
+landing_speed = Symbol("landing_speed", units.velocity)
+
+E_k = kinetic_energy.law.subs({
+    kinetic_energy.body_mass: body_mass,
+    kinetic_energy.body_velocity: landing_speed
+})
+E_p = potential_energy.law.subs({
+    potential_energy.body_mass: body_mass,
+    potential_energy.height: height,
+    potential_energy.free_fall_acceleration: gravity_acceleration
+})
+
+law = [E_p, E_k]
+
+solved = solve(law, (landing_speed, kinetic_energy.kinetic_energy_of_body), dict=True)[0][kinetic_energy.body_velocity]
```

Comment:
What if we assume so? The increase in kinetic energy is due to a change in the potential energy of the gravity field
$E_k - E_{k0} = E_p - E_{p0}$.
That is
$(m/2)(v^2 - v_0^2) = mg (h_0 - h)$,
where
$v_0$ - initial velocity of fall (by condition equal to zero),
$v$ is the final velocity before landing,
$h_0$ is the initial height,
$h$ is the final height (we assume at the moment before landing $h=0$).

Reply:
I think it is easier to add conservation condition:
conservation_law_applied = conservation_law.subs(E(t1), potential_energy).subs(E(t2), kinetic_energy)
law = [E_p, E_k, conservation_law_applied]

Reply:
First law getting from sistem:
1. $E_{k0} + E_{p0} = E$, because in start moment $E_{k_0} = 0$ and $E_{p0} = mgh_0$;
2. $E_k + E_p = E$, because before landing $E_k = mv^2/2$ and $E_p = mgh = 0$

Then 
$E_{k0} + E_{p0} = E_k + E_p$
And we has
$E_{p0} - E_p = E_k - E_{k0}$

Reply:
Why do you want to subtract them instead of adding?

Reply:
> Я думаю, что проще добавить условие сохранения: Conservation_law_applied = Conservation_law.subs(E(t1), потенциально_энергия).subs(E(t2), кинетическая_энергия) Law = [E_p, E_k, Conservation_law_applied]

I will try this.

Reply:
BTW we avoid any implicit laws, so we prefer not to subtract or add anything in the examples, but use law for mechanical energy: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/definitions/mechanical_energy_is_kinetic_and_potential.py

Reply:
Thanks! How to substitute one formula into another using code? For example, in mechanical energy
$E = K + P$
i will paste
$K = mv^2/2$
And when i extract v,
```python
solved = solve(law, landing_speed, dict=True)[0][landing_speed]
```
i got
$v = \sqrt{2} * \sqrt{K/m}$

Reply:
> Почему вы хотите их вычитать, а не добавлять?

You're right, i agree. It is not necessary to separate kinetic and potential energies in equation $E_{k0} + E_{p0} = E_k + E_p$. And two terms turn out to be equal to zero in the continuation of the solution.

Reply:
You have Ep_before, Ek_before, Ep_after, Ek_after.
You can do it this way:
mechanical_energy_applied_before = mechanical_energy.law.subs({mechanical_energy.K: Ek_before, mechanical_energy.P: Ep_before})

Also to avoid having too many laws in one solve, you can save intermediate results, eg `mechanical_energy_before = mechanical_energy_applied_before.rhs`

Reply:
In the dictionary, what should be submitted as a key? An instance of the 'Symbol' class is entered as the value, if I understand correctly.

Reply:
It's a symbol to substitute. Value is what we are substituting with.